### PR TITLE
[SVA-271] Remove emojis from content_blocks

### PR DESCRIPTION
**Pull request by tonekk**
[SVA-271](https://tpwd.atlassian.net/browse/SVA-271)

### CODE REVIEW

File: spec/factories/news_items.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@
     publication_date { "2019-05-02 12:50:44" }
     published_at { "2019-05-02 12:50:44" }
     show_publish_date { false }
+    data_provider
```
Comment (donni106) [inline spec/factories/news_items.rb:12]: why was this added here?

File: spec/models/content_block_spec.rb [hunk 2]
```diff
@@ -5,6 +5,20 @@
...
   it { is_expected.to have_many(:media_contents) }
+
+  ["title", "intro", "body"].each do |column|
+    it "should remove emojis from #{column} before saving" do
+      content_block =  FactoryBot.build(:content_block, {
+        intro: "test😊😍😌🤕👿👹👧👧🏻👧🏼👧🏽🏼👍🏽👌☝🏼🥝🥦🌶🌽🍎",
+        title: "test😊😍😌🤕👿👹👧👧🏻👧🏼👧🏽🏼👍🏽👌☝🏼🥝🥦🌶🌽🍎",
+        body: "test😊😍😌🤕👿👹👧👧🏻👧🏼👧🏽🏼👍🏽👌☝🏼🥝🥦🌶🌽🍎",
+        content_blockable: FactoryBot.create(:news_item)
+      })
```
Comment (donni106) [inline spec/models/content_block_spec.rb:16]: wouldn't something like the following work to reduce the triple same stuff?

```
content_block =  FactoryBot.build(:content_block, {
  [column]: "test😊😍😌🤕👿👹👧👧🏻👧🏼👧🏽🏼👍🏽👌☝🏼🥝🥦🌶🌽🍎",
  content_blockable: FactoryBot.create(:news_item)
})
```

but did not work out for me testing it this way

**Review by donni106 — APPROVED** [review]

File: spec/models/content_block_spec.rb (hunk 2 above)
Comment (tonekk) [reply spec/models/content_block_spec.rb]: Fixed in ff76324
